### PR TITLE
My hw 05 fixed

### PR DESCRIPTION
**Pull request by hegoru**

### CODE REVIEW

File: HW03/egor_usik/egor_usik_hw_03_t_02.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative 'expressions'
+
+class Homework3
+  # def task2(text)
```
Comment (AliakseiAndryianau) [inline HW03/egor_usik/egor_usik_hw_03_t_02.rb:6]: remove this commented code please

File: HW03/egor_usik/egor_usik_hw_03_t_03.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'date'
+require_relative 'expressions'
+
+class Homework3
+  # def task3(text)
```
Comment (AliakseiAndryianau) [inline HW03/egor_usik/egor_usik_hw_03_t_03.rb:7]: remove comments please

File: HW04/egor_usik/spec/lib/person_spec.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe Person do
+  subject(:create) { described_class.new(name: 'John', surname: 'Doe') }
```
Comment (AliakseiAndryianau) [inline HW04/egor_usik/spec/lib/person_spec.rb:5]: use just subject keyword

File: HW04/egor_usik/spec/lib/person_spec.rb [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      end
+    end
+  end
+
+  describe '#mark_as_read!' do
+    context 'when subject reads all notifications' do
+      let(:notification_class) { Notification.new('Test', 'Test notification') }
+
+      it 'returns an empty array' do
+        # notification = instance_double("Notification", :title => 'Test', :description => 'Test notification')
```
Comment (AliakseiAndryianau) [inline HW04/egor_usik/spec/lib/person_spec.rb:45]: remove comment please

File: HW04/egor_usik/spec/lib/person_spec.rb [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        create.send(:add_notification, notification_class)
```
Comment (AliakseiAndryianau) [inline HW04/egor_usik/spec/lib/person_spec.rb:46]: just call methods by name without .send

File: HW04/egor_usik/spec/lib/student_spec.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe Student do
+  subject(:create) { described_class.new(name: 'Sam', surname: 'Smith') }
```
Comment (AliakseiAndryianau) [inline HW04/egor_usik/spec/lib/student_spec.rb:4]: just a subject please

File: HW03/egor_usik/egor_usik_hw_03_t_02.rb (hunk 1 above)
Comment (hegoru) [reply HW03/egor_usik/egor_usik_hw_03_t_02.rb]: Done

File: HW03/egor_usik/egor_usik_hw_03_t_03.rb (hunk 2 above)
Comment (hegoru) [reply HW03/egor_usik/egor_usik_hw_03_t_03.rb]: Comment was removed

File: HW04/egor_usik/spec/lib/person_spec.rb (hunk 3 above)
Comment (hegoru) [reply HW04/egor_usik/spec/lib/person_spec.rb:7]: Fixed that

File: HW04/egor_usik/spec/lib/person_spec.rb (hunk 4 above)
Comment (hegoru) [reply HW04/egor_usik/spec/lib/person_spec.rb:47]: Comment removed

**Pull request merged**